app/script/view_model/content/TitleBarViewModel.js

```diff
@@ -109,6 +114,22 @@ z.viewModel.content.TitleBarViewModel = class TitleBarViewModel {
     this.showDetails();
   }
 
+  onMousedown() {
+    this.isDragged = this.isMacDesktop;
+  }
+
+  onMousemove() {
+    this.isMoved = this.isDragged;
+  }
+
+  onMouseup() {
```

Comment:
`onMouseUp`, `onMouseMove`, `onMouseDown`

Reply:
the events are called all lowercase "mousedown", "mousemove" and "mouseup", so I decided to stick with those.

Reply:
I think it would still look better.. :)